mlflow/openai/_openai_autolog.py

```diff
@@ -0,0 +1,224 @@
+import inspect
+import json
+import logging
+import os
+import uuid
+from contextlib import contextmanager
+from copy import deepcopy
+from typing import Iterator
+
+from packaging.version import Version
+
+import mlflow
+from mlflow import MlflowException
+from mlflow.entities import RunTag
+from mlflow.ml_package_versions import _ML_PACKAGE_VERSIONS
+from mlflow.tracking.context import registry as context_registry
+from mlflow.tracking.fluent import _get_experiment_id
+from mlflow.utils.autologging_utils import disable_autologging, get_autologging_config
+from mlflow.utils.autologging_utils.safety import _resolve_extra_tags
+
+MIN_REQ_VERSION = Version(_ML_PACKAGE_VERSIONS["openai"]["autologging"]["minimum"])
+MAX_REQ_VERSION = Version(_ML_PACKAGE_VERSIONS["openai"]["autologging"]["maximum"])
+
+_logger = logging.getLogger(__name__)
+
+
+def _get_input_from_model(model, kwargs):
+    from openai.resources.chat.completions import Completions as ChatCompletions
+    from openai.resources.completions import Completions
+    from openai.resources.embeddings import Embeddings
+
+    model_class_param_name_mapping = {
+        ChatCompletions: "messages",
+        Completions: "prompt",
+        Embeddings: "input",
+    }
+    if param_name := model_class_param_name_mapping.get(model.__class__):
+        # openai tasks accept only keyword arguments
+        if param := kwargs.get(param_name):
+            return param
+        input_example_exc = MlflowException(
+            "Inference function signature changes, please contact MLflow team to "
+            "fix OpenAI autologging.",
+        )
+    else:
+        input_example_exc = MlflowException(
+            f"Unsupported OpenAI task. Only support {list(model_class_param_name_mapping.keys())}."
```

Comment:
```suggestion
            f"Unsupported OpenAI task. Only support {[c.__name__ for c in model_class_param_name_mapping]}."
```

Reply:
chat completions and completions both have same class name `Completions`, so it will be a bit confuing.

I updated it as a hardcoded plain text, this should be easier